--- e1000e-1.1.2/src/ethtool.c~	2009-10-31 01:02:06.000000000 +0100
+++ e1000e-1.1.2/src/ethtool.c	2010-03-07 19:01:52.167295997 +0100
@@ -2036,8 +2036,10 @@
 	.get_strings		= e1000_get_strings,
 	.phys_id		= e1000_phys_id,
 	.get_ethtool_stats	= e1000_get_ethtool_stats,
+#if ( LINUX_VERSION_CODE < KERNEL_VERSION(2,6,33) )
 	.self_test_count	= e1000_get_self_test_count,
 	.get_stats_count	= e1000_get_stats_count,
+#endif
 	.get_coalesce		= e1000_get_coalesce,
 	.set_coalesce		= e1000_set_coalesce,
 #ifdef NETIF_F_LRO
